docs/graphql/manual/api-reference/schema-metadata-api/relationship.rst

```diff
@@ -453,3 +453,52 @@ Args syntax
      - false
      - Text
      - Comment
+
+.. _rename_relationship:
+
+rename_relationship
+-------------------
+
+``rename_relationship`` is used to the modify name of an existing relationship.
+
+An example:
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "type": "rename_relationship",
+       "args": {
+           "table": "article",
+           "name": "article_details",
+           "new_name": "article_detail"
+       }
+   }
+
+.. _rename_relationship_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - table
+     - true
+     - :ref:`TableName`
+     - Name of the table
+   * - name
```

Comment:
We just approved a PR where the `name` field here was changed to `relationship`: https://github.com/hasura/graphql-engine/pull/3419. I think this needs to be changed here too. Correct, @rakeshkky ?

Reply:
No @marionschleifer, This is renaming a relationship. So we have `name` and `new_name` fields.

Reply:
Okay thanks for letting me know @rakeshkky 👍